tests/mod.rs

```diff
@@ -50,6 +53,8 @@ fn assert_dirs_eq(expected: &Path, actual: &Path) {
             .read_to_string(&mut created)
             .expect("Could not read to string");
 
+        let created = String::from_iter(normalized(created.chars()));
```

Comment:
if `.gitattributes` is saying `lf` for everything, and the test should be generating `lf` based output, why is normalization needed here?

Reply:
uh, good point, I… don't know, and without, it does not work so it seems I failed here… need more work :-/

Reply:
I forgot one normalization when reading the layout, it makes the situation better but my test still not pass :-/

Reply:
Found multiples occurences of `read_file` some might need normalization as well, I'll see that later